src/sentry/static/sentry/app/views/projectFilters.jsx

```diff
@@ -297,19 +309,58 @@ const ProjectFiltersSettingsForm = React.createClass({
     );
   },
 
+  renderAdditionalFilters() {
+    let errors = this.state.errors;
+    return (
+      <div>
+        <h5>{t('Filter errors from these releases:')}</h5>
+        <TextareaField
+          key="release"
+          name="release"
+          help={this.renderLinkToGlobWiki()}
+          placeholder="e.g. 1.* or [!3].[0-9].*"
+          value={this.state.formData['filters:releases']}
+          error={errors['filters:releases']}
+          onChange={this.onFieldChange.bind(this, 'filters:releases')}
+        />
+        <h5>{t('Filter errors by error message:')}</h5>
+        <TextareaField
+          key="errorMessage"
+          name="errorMessage"
+          help={this.renderLinkToGlobWiki()}
+          placeholder="e.g. TypeError* or *: integer division or modulo by zero"
+          value={this.state.formData['filters:error_messages']}
+          error={errors['filters:error_messages']}
+          onChange={this.onFieldChange.bind(this, 'filters:error_messages')}
+        />
+      </div>
+    );
+  },
+
+  renderDisabledFeature() {
+    let project = this.getProject();
+    let organization = this.getOrganization();
+    return this.state.hooksDisabled
```

Comment:
I have no context here, but do we need to iterate through all of `hooksDisabled` here since we're calling `shift()` afterwards (unless `hook()` is producing a side effect)

Reply:
No, it seems like I could do something like 
`return this.state.hooksDisabled[0](organization, project);`
since this is the only component returned with `'project:custom-inbound-filters:disabled'` and this.state.hooksDisabled in this case will only have 1 item. Do you think that would be better?

Reply:
Ah, I see what that's doing now. I think it's better to not assume it will always have 1 item, can we remove the `shift()` and return the entire array?